specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -1177,9 +1193,13 @@
     "MoveResourceProperties": {
       "description": "Defines the move resource properties.",
       "type": "object",
+      "required": [
+        "sourceId"
+      ],
       "properties": {
         "provisioningState": {
-          "$ref": "#/definitions/ProvisioningState"
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
```

Comment:
This will not work. If you want to modify the readOnly, please consider adding another model

Reply:
do you mean this is breaking change and hence not allowed to change it to readonly?

Reply:
I mean it will not work to add "readOnly" on a ref directly. Only comment and description are allowed on ref

Reply:
got it.